lib/github_service/commands/cross_repo_test.rb

```diff
@@ -173,9 +175,14 @@ def expand_repo_group(repo_group)
       end
 
       def normalize_repo_name(repo)
+        repo = repo.strip
         repo.include?("/") ? repo : "#{issue.organization_name}/#{repo}"
       end
 
+      def branch_or_pr?(repo)
```

Comment:
Seems like this method should have been added in the next commit... instead of part of this one.

Reply:
Oh yeah you're right...I missed that on the rebase -i